### PR TITLE
dev to release

### PR DESCRIPTION
**Pull request by sdolenc**
dev to release: MSA ui, image with broader browser support, and cookie re-consent (which is already in master/PROD, but not in release/INT yet), security: disable autocomplete on sensitive form fields (like passwords, etc)

### CODE REVIEW

**Comment by sdolenc** [comment]
@satyarapelly , are these MSA changes safe to deploy without any corresponding edx-platform changes or should we selectively hold them back?

**Review by kabirkhan — APPROVED** [review]

**Review by btelnes — APPROVED** [review]

File: comprehensive/lms/templates/navigation.html [hunk 1]
```diff
@@ -64,7 +64,7 @@ <h1 class="logo">
         </%block>
       </a>
     </h1>
-
+    % if not enable_account_linking:
```
Comment (kevsers) [inline comprehensive/lms/templates/navigation.html:67]: we should probably leave this change out in favor of not mixing in MSA changes..
Also I am not quite sure if we have that setting available already.. we might need to check oxa-tools.

**Review by kevsers — APPROVED** [review]

**Comment by sdolenc** [comment]
The MSA changes in here require corresponding edx-platform changes for MSA. I'll submit a partial dev-to-release PR for edx-theme with MSA changes excluded

**Comment by sdolenc** [comment]
the non-MSA change are already in release. There's no need to submit a new PR for edx-theme at this time.

**Pull request closed without merging**